pkg/workflows/sdk/v2/pb/sdk.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package cre.sdk.v2;
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb";
+
+import "google/protobuf/any.proto";
+import "values/pb/values.proto";
+
+message CapabilityRequest {
```

Comment:
Can we use protos from Capability API directly? (pkg/capabilities/pb/capabilities.proto)

Reply:
No, it leaks metadata and old values.Value fields that the workflow author doesn't fill in. I want a clean separation of what the workflow author can do (even if it's inside the SDK) and what the host or engine will do.